ALAuto.py

```diff
@@ -200,11 +202,31 @@ def print_cycle_stats(self):
 
 try:
     while True:
-        Utils.update_screen()
+        Utils.wait_update_screen(1)
 
         # temporal solution to event alerts
-        if not Utils.find("menu/button_battle"):
-            Utils.touch_randomly(Region(54, 57, 67, 67))
+        # if not Utils.find("menu/button_battle"):
+        #     Utils.touch_randomly(Region(54, 57, 67, 67))
+        #     Utils.script_sleep(1)
+        #     continue
+        if Utils.find("menu/item_found"):
+            Logger.log_msg("Found item message.")
+            Utils.find_and_touch("menu/tap_to_continue")
+            Utils.script_sleep(1)
+            continue
+        if Utils.find("menu/home_button"):
+            Logger.log_msg("Found home button")
+            Utils.find_and_touch("menu/home_button")
+            Utils.script_sleep(1)
+            continue
+        if Utils.find("menu/announcement"):
+            Logger.log_msg("Found Announcement Window")
+            Utils.find_and_touch("menu/announcement")
+            Utils.script_sleep(1)
+            continue
+        if Utils.find("menu/alert_info"):
+            Logger.log_msg("Found alert.")
+            Utils.find_and_touch("menu/alert_close")
```

Comment:
Please avoid formatting changes, I like how the current structure is and PRs that change it are instantly a big no no.

Regarding your addition, though: the code should be fine and it is not a bad implementation per se, but I'd like to test any changes to the main file myself. Also the fact that this notification could pop up while other modules are running makes this solution impractical.

Reply:
The purpose of my implementation is just to keep the bot running on the commission module, I see no reason to work with other modules.